src/transformers/models/realm/retrieval_realm.py

```diff
@@ -115,7 +117,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             )
         block_records = np.load(block_records_path, allow_pickle=True)
 
-        tokenizer = RealmTokenizer.from_pretrained(pretrained_model_name_or_path, *init_inputs, **kwargs)
```

Comment:
Let's maybe use `AutoTokenizer` here so that the user can decide between Fast and Non-Fast